### PR TITLE
xytraceset in quicklook

### PR DESCRIPTION
**Pull request by julienguy**
Replace all instances of quicklook.qlpsf.PSF by in quicklook , modify read_xytraceset and add write_xytraceset.
This PR fixes the issue #681 .

### CODE REVIEW

**Comment by julienguy** [comment]
I realize my description of the pull request was not intelligible, sorry for that. Let me try again:
- I replaced everywhere in quicklook the use of the class qlpsf.PSF by the class desispec.xytraceset which is used in the offline pipeline. xytraceset is a container for the trace coordinates and the PSF sigma values.
- I added write_xytraceset in desispec.io.xytraceset
- This is to make sure we use only one set of IO routines for spectral traces in desispec, in order to avoid inconsistencies as reported in issue #681.
- This turned out to touch a lot of files in quicklook.

**Comment by rstaten** [comment]
I have tested this branch on arcs and science and no longer receive the errors we were previously seeing. Thanks for this fix.

**Comment by sbailey** [comment]
Thanks for testing @rstaten.  Merging now.
